packages/components/src/components/LoadingIndicator/LoadingIndicator.stories.tsx

```diff
@@ -0,0 +1,59 @@
+import { Meta, StoryObj } from '@storybook/react'
+import { View } from 'react-native'
+import React from 'react'
+
+import { Link } from '../Link/Link'
+import { LoadingIndicator, LoadingIndicatorProps } from './LoadingIndicator'
+import { Spacer } from '../Spacer/Spacer'
+import { generateDocs } from '../../utils/storybook'
+
+const meta: Meta<LoadingIndicatorProps> = {
+  title: 'LoadingIndicator',
```

Comment:
I'd think this should be the external facing name per Figma:  `Loading indicator`.

Reply:
Reverted. Wasn't aware we were using the figma naming in our stories